fe/fe-core/src/main/java/com/starrocks/statistic/StatisticUtils.java

```diff
@@ -208,6 +219,24 @@ public static void triggerCollectionOnFirstLoad(TransactionState txnState, Datab
         }
 
         if (sync) {
+            // wait a short-time for the task getting executed, if too many jobs in the queue we just give up
+            // waiting, otherwise it may block the data loading for a long period
+            try {
+                future.get(1, TimeUnit.SECONDS);
+                Preconditions.checkState(isRunning.get());
+                return;
+            } catch (InterruptedException | ExecutionException e) {
+                LOG.error("failed to execute statistic collect job", e);
+            } catch (TimeoutException e) {
+                if (!isRunning.get()) {
+                    LOG.warn("await collect statistic task failed after 1 seconds, which mean too many jobs in the " +
+                            "queue");
+                    return;
+                }
+            }
```

Comment:
so...there is already support `semi_sync_collect_statistic_await_seconds`, if wait is intolerable, why not use the async mode?